src/javascript/app/base/binary_loader.js

```diff
@@ -149,7 +149,7 @@ const BinaryLoader = (() => {
             return;
         }
 
-        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] });
+        const div_container = createElement('div', { class: 'logged_out_title_container', html: content.getElementsByTagName('h1')[0] ? content.getElementsByTagName('h1')[0] : '' });
```

Comment:
better store it in a variable so we don't need to retrieve the element twice

Reply:
or use `content.getElementsByTagName('h1')[0] || ''`

Reply:
Okay 👍